### ISSUE TITLE
Unable to open GITHUB SETTINGS from dgit

### ISSUE THREAD

**Issue by YousraMashkoor** [bunsenstraat/remix-storage-plugin#43]
A warning shows up when clicking on GITHUB SETTINGS, clicking on the `check settings` button performs no action
![image](https://user-images.githubusercontent.com/37848888/188512340-2cc43d31-b734-4293-b66b-0d9563d1177a.png)

**Comment by 6str** [comment]
unable to change my token. the settings tab gives me the green light that the token is set up, but the problem is it is expired and I now need to change it. I can't see the option to do that anymore.

<img width="128" alt="image" src="https://user-images.githubusercontent.com/56482955/189493570-298ca20b-cccb-422b-b586-834db2382f87.png">

**Comment by ryestew** [comment]
I was about to make this issue, but it has already been made.

**Comment by adipto** [comment]
You have to create a access key in git hub then copy that key in remix setting, it will also ask you for github username and email

**Comment by YousraMashkoor** [comment]
> You have to create a access key in git hub then copy that key in remix setting, it will also ask you for github username and email

yes, but when I'm clicking on it to add the username and email it's not letting me do it. Other setting options are working fine.

**Comment by bunsenstraat** [comment]
where are you adding the username email? @YousraMashkoor

**Comment by bunsenstraat** [comment]
![Screenshot 2022-09-19 at 11 48 31](https://user-images.githubusercontent.com/2787300/190992021-c8ad55f1-2371-4424-a5a7-a1aad5fdf012.png)

**Comment by bunsenstraat** [comment]
check settings just checks the settings, doesn't open the settings menu. You have to do that yourself.
